clickhouse-client/src/test/java/com/clickhouse/client/ClientIntegrationTest.java

```diff
@@ -2599,6 +2674,7 @@ public void testLoadBalancingPolicyFailover(ClickHouseLoadBalancingPolicy loadBa
 
     @Test(groups = {"integration"})
     public void testFailover() throws ClickHouseException {
+        if (isCloud()) return; //TODO: testFailover - Revisit
```

Comment:
Can we add a link to https://github.com/ClickHouse/clickhouse-java/issues/1747 ? It would help us to search for these disabled tests. Even though, I don't think we're going to fix all of them

Reply:
@Paultagoras I've found 106 `TODO` in the project. Without a link, it is hard to understand when and how these are going to be addressed (if ever)